understanding/unusual-words.html

```diff
@@ -58,133 +56,115 @@
       <div class="default-grid with-gap leftcol">
          <aside class="box nav-hack sidebar standalone-resource__sidebar ">
             <nav>
-               <header class="box-h ">Page Contents</header>
+               <header class="box-h ">このページの内容</header>
                <div class="box-i">
                   <ul>
-                     <li><a href="#brief">In Brief</a></li>
-                     <li><a href="#intent">Intent</a></li>
-                     <li><a href="#benefits">Benefits</a></li>
-                     <li><a href="#examples">Examples</a></li>
-                     <li><a href="#resources">Related Resources</a></li>
-                     <li><a href="#techniques">Techniques</a></li>
-                     <li><a href="#key-terms">Key Terms</a></li>
+                     <li><a href="#brief">要約</a></li>
+                     <li><a href="#intent">意図</a></li>
+                     <li><a href="#benefits">利点</a></li>
+                     <li><a href="#examples">事例</a></li>
+                     <li><a href="#resources">関連リソース</a></li>
+                     <li><a href="#techniques">テクニック</a></li>
+                     <li><a href="#key-terms">重要な用語</a></li>
                   </ul>
                </div>
             </nav>
          </aside>
          <main id="main" class="standalone-resource__main">
 
-         <!-- 日本語訳されるまでの一時的な注記 -->
-         <div class="untranslated-note"><p>このWCAG 2.2 解説書の日本語訳は作業中となっています。WCAG 2.1 解説書の日本語訳をご利用いただけます。<br>[ <a href="https://waic.jp/translations/WCAG21/Understanding/">WCAG 2.1 解説書</a> ]</p></div>
-            <h1><span class="standalone-resource__type-of-guidance">Understanding 
-                  				SC 3.1.3:</span>Unusual Words (Level AAA)
+            <h1><span class="standalone-resource__type-of-guidance">解説書 
+                  				達成基準 3.1.3:</span>一般的ではない用語 (レベル AAA)
             </h1>
             <aside class="box">
-               <header class="box-h  box-h-icon">Success Criterion (SC) </header>
+               <header class="box-h  box-h-icon">達成基準</header>
                <div class="box-i">
-                  <p>A <a href="#dfn-mechanism">mechanism</a> is available for identifying specific definitions of words or phrases <a href="#dfn-used-in-an-unusual-or-restricted-way">used in an unusual or restricted way</a>, including <a href="#dfn-idiom">idioms</a> and <a href="#dfn-jargon">jargon</a>.
+                  <p><a href="#dfn-idiom">慣用句</a>及び<a href="#dfn-jargon">専門用語</a>を含めて、<a href="#dfn-used-in-an-unusual-or-restricted-way">一般的ではない、又は限定された用法で使われている</a>単語、又は語句の、明確な定義を特定する<a href="#dfn-mechanism">メカニズム</a>が利用できる。
```

Comment:
「専門用語」→「業界用語」かと思います。

```suggestion
                  <p><a href="#dfn-idiom">慣用句</a>及び<a href="#dfn-jargon">業界用語</a>を含めて、<a href="#dfn-used-in-an-unusual-or-restricted-way">一般的ではない、又は限定された用法で使われている</a>単語、又は語句の、明確な定義を特定する<a href="#dfn-mechanism">メカニズム</a>が利用できる。
```